src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -61,8 +61,13 @@ public void ExecuteAnalysis_RequestCannotBeCreated_NoAnalysis()
             var testSubject = CreateTestableAnalyzer(requestFactory: requestFactory.Object);
             testSubject.ExecuteAnalysis("path", "charset", new[] { AnalysisLanguage.CFamily }, Mock.Of<IIssueConsumer>(), analysisOptions, CancellationToken.None);
 
-            testSubject.TriggerAnalysisCallCount.Should().Be(0);
+            testSubject.WaitForFactoryCallToComplete();
```

Comment:
I think it would be easier to read the tests if we were to call the internal async method.